python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -770,8 +841,9 @@ def targets_unresolved_asset_job(self) -> bool:
     def load_target(
         self,
     ) -> Union[GraphDefinition, PipelineDefinition, UnresolvedAssetJobDefinition]:
-        if isinstance(self._target, DirectTarget):
-            return self._target.load()
+        target = self._target
```

Comment:
This is necessary for type-checking?

Reply:
nope, I think this was just a remnant of me experimenting with the return types. will remove